packages/devtools_app/lib/src/inspector/inspector_tree.dart

```diff
@@ -408,6 +412,7 @@ abstract class InspectorTreeController {
 
   InspectorTreeNode get root => _root;
   InspectorTreeNode _root;
+
```

Comment:
remove this extra blank line. the recommended pattern is to keep the getter, setter and private field name in a group.